content/knowledge-base/migrating-from-bitrise.md

```diff
@@ -0,0 +1,151 @@
+---
+description: How to migrate from Bitrise to Codemagic
+title: Migrating from Bitrise
+weight: 13
+---
+
+Codemagic makes use of `codemagic.yaml` for configuring your workflow. There is also a Flutter workflow editor for Flutter applications, which simplifies the setup but removes some flexibility.
+
+## Migrating iOS builds with codemagic.yaml
+
+If you have already set up your iOS application on Bitrise, migrating to Codemagic is straightforward.
```

Comment:
Can we make a comparison like [here](https://circleci.com/docs/2.0/migrating-from-aws/#configuration-comparison)?